### ISSUE TITLE
[Feature Request] Surround selected text with formatting characters

### ISSUE THREAD

**Issue by Nabav** [idleberg/vscode-textile#10]
It would be great, if the selected text is surrounded with the following formatting characters instead of simply being replaced:

* `@`: monospace font
* `"`: linked text
* `*`: bold text
* `_`: italic text
* `-`: striked-though text
* `(`: block of text with `)`
* `[`: block of text with `]`
* `{`: block of text with `}`

This feature is there for Markdown. Single-quote and backticks can be added as well.

**Comment by Nabav** [comment]
@idleberg, it seems to be really simple. Please have a look here: https://code.visualstudio.com/api/language-extensions/language-configuration-guide#autosurrounding

**Comment by Nabav** [comment]
@idleberg, I have changed `textile.configuration.json` to the code below and everything works.

```json
{
    "comments": {
        "blockComment": [ "<!--", "-->" ]
    },
    "brackets": [
        ["{", "}"],
        ["[", "]"],
        ["(", ")"]
    ],
    "surroundingPairs": [
        ["{", "}"],
        ["[", "]"],
        ["(", ")"],
        ["'", "'"],
        ["\"", "\""],
        ["`", "`"],
        ["@", "@"],
        ["*", "*"],
        ["_", "_"],
        ["-", "-"]
    ]
}
```

**Comment by idleberg** [comment]
Thanks @Nabav for looking into it! Do you want to submit a PR? I can merge it on Monday or add the changes myself, if you prefer.

**Comment by idleberg** [comment]
Closed in v0.2.0. Thanks again!
